2025-08-18 08:51:05 [INFO] tests.request.test_request_api: Тест: Внесение вклада в несуществующий запрос (POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution) [test_request_api.py:220]
2025-08-18 08:51:05 [INFO] api.request.client: Вызов POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [client.py:79]
2025-08-18 08:51:05 [INFO] core.http_client: Sending POST request to /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:73]
2025-08-18 08:51:05 [INFO] core.http_client: Received response 500 from http://localhost:4040/api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:80]
2025-08-18 08:51:05 [WARNING] utils.allure_utils: Ответ не является валидным JSON, аттачим как текст. [allure_utils.py:62]
2025-08-18 08:51:05 [INFO] tests.request.test_request_api: Тест: Внесение вклада в несуществующий запрос (POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution) [test_request_api.py:220]
2025-08-18 08:51:05 [INFO] api.request.client: Вызов POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [client.py:79]
2025-08-18 08:51:05 [INFO] core.http_client: Sending POST request to /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:73]
2025-08-18 08:51:05 [INFO] core.http_client: Received response 403 from http://localhost:4040/api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:80]
2025-08-18 08:51:05 [INFO] tests.request.test_request_api: Тест: Внесение вклада в несуществующий запрос (POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution) [test_request_api.py:220]
2025-08-18 08:51:05 [INFO] api.request.client: Вызов POST /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [client.py:79]
2025-08-18 08:51:05 [INFO] core.http_client: Sending POST request to /api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:73]
2025-08-18 08:51:05 [INFO] core.http_client: Received response 403 from http://localhost:4040/api/request/non-existent-e74d0ee4-03b7-43b9-9536-ccceab04456c/contribution [http_client.py:80]
